README.md

```diff
@@ -1,2 +1,15 @@
 # mobile-automation
 Mobile automation using appium framework - Beginners
+
+Github
+1) Navigate to www.github.com
+2) Create a new account
+3) After creating a new account, navigate to the following link: https://github.com/HarshBhardwaj/mobile-automation
+4) Click "fork" in top right (this will copy the project to your account)
+
+Git Bash
+1) Navigate to https://git-scm.com/downloads
+2) Down load the git bash for your operating system
+3) Open git bash
+4) "cd" to your project folder where you want to clone the repo
+5) Execute: git clone (url of the "fork" repo you did above)
```

Comment:
Need to add steps for sync setup of forked repo with original.

Reply:
I will create a wiki document for those steps or provide links in the document to videos that show setups.

Reply:
Cool, thanks!